memegen/templates/share.html

```diff
@@ -27,12 +27,12 @@
         </div>
         <br>
         <div class="row text-center">
-          <a class="btn btn-primary disable-link" href="https://www.facebook.com/sharer/sharer.php?u={{ url }}" target="_blank">
+          <a class="btn btn-primary disable-link" href="https://www.facebook.com/sharer/sharer.php?u={{ url.replace('?', '&') | safe }}" target="_blank">
```

Comment:
It sounds like this won't work because Facebook strips the extra query parameter.